<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulkConversionDialog</class>
 <widget class="QDialog" name="BulkConversionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Conversion</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="selectLabel">
     <property name="text">
      <string>Select Files </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="browsePushButton">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="filenameListWidget">
     <property name="toolTip">
      <string>Press delete key with filenames selected to remove them from the list.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="targetLabel">
     <property name="text">
      <string>Target Directory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="targetLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="targetPushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="inputFormatLabel">
     <property name="text">
      <string>Input Format</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="inputComboBox">
     <item>
      <property name="text">
       <string>Wide Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Long Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vespucci Binary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Old Vespucci Binary</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="outputFormatLabel">
     <property name="text">
      <string>Output Format</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="outputComboBox">
     <item>
      <property name="text">
       <string>Vespucci Binary</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="swapCheckBox">
     <property name="text">
      <string>Swap x and y?</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BulkConversionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BulkConversionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
